tools/cbmc/proofs/Queue/QueueCreateCountingSemaphore/QueueCreateCountingSemaphore_harness.c

```diff
@@ -0,0 +1,16 @@
+#include "FreeRTOS.h"
+#include "queue.h"
+
+#include "cbmc.h"
+
+
+void harness(){
+    UBaseType_t uxMaxCount;
+    UBaseType_t uxInitialCount;
+
+    __CPROVER_assume(uxMaxCount != 0);
```

Comment:
Why?

Reply:
Because it is an explicit assert in the implementation that will otherwise be triggered.